assignments/a6.md

```diff
@@ -17,22 +17,27 @@ This is the final project for the course. You will deliver a functioning proof o
 
 1. Code
    - All code should be easily accessible in repos under the DCSIL organization
-   - All repos should be linked to your team in the DCSIL Team App (**TODO** Add link to docs?)
+   - All repos should be linked to your team in the DCSIL Team App (Add a new repo [here](https://dcsil-team-app.herokuapp.com/my_team/repos/new).)
+   - All of your repos have been added to [Code Climate](https://codeclimate.com/). Your repo should maintain a minimum "B" average in "maintainability".
```

Comment:
Will you be providing a lecture of some sorts about Code Climate or some sort of information about what things it looks at? It might be good for us to provide some guidance or insight here so that it doesn't feel like a black box to the students.

Reply:
I added this doc: https://dcsil-team-app.herokuapp.com/docs/software/code/code_climate
How does it look to you?

Reply:
Okay yeah this is definitely more clear and transparent about what exactly students can expect. Thanks!!